### ISSUE TITLE
Koop Adapter: GeoRSS provider

### ISSUE THREAD

**Issue by ajturner** [koopjs/koop#106]
It would be useful to have a Koop adapter for GeoRSS feeds.

For example: http://maps.cmpd.org/datafeeds/accidentsgeorss.ashx

**Comment by slibby** [comment]
@ajturner this is superseded by a newer issue with more detail/discussion, close?

**Comment by chelm** [comment]
@slibby Yeah it was brought up yesterday - https://github.com/Esri/koop/issues/185

@ajturner replied in an email that it might make more sense to create a custom layer in the JSAPI that could render GeoRSS (if the motivation is purely just to render a georss layer on a map)

**Comment by astauffer** [comment]
FYI... @ajturner you can add GeoRSS to a web map now, and I believe change the render.

**Comment by ajturner** [comment]
@astauffer and it's all client side?

I saw that upcoming WFS layer is client side - very similar XML, so hopefully that's the case.

**Comment by astauffer** [comment]
@ajturner http://doc.arcgis.com/en/arcgis-online/reference/georss.htm

**Comment by ajturner** [comment]
Thanks @astauffer that confirms what I was saying. The current GeoRSS implementation requires reflecting to a Server-side query to convert the XML into a feature collection. I think this was the instigation for @mhogeweg [question](https://github.com/Esri/koop/issues/185) 

If you watch the network log of the example

`http://utility.arcgis.com/sharing/rss?url=http%3A%2F%2Fmaps.cmpd.org%2Fdatafeeds%2Faccidentsgeorss.ashx%2F&outSR=%7B%22wkid%22%3A102100%2C%22latestWkid%22%3A3857%7D&callback=dojo.io.script.jsonp_dojoIoScript1._jsonpCallback`

![arcgis_-_georss_layer](https://cloud.githubusercontent.com/assets/1218/8107535/f3ca83b4-1017-11e5-93ff-0fd67425b04c.png)

/cc @jbartley

**Comment by astauffer** [comment]
If the georss needs to hit utlity as a proxy, wouldn't a georss provider for koop essentially do the exact same thing?

**Comment by ajturner** [comment]
Indeed - we are mixing streams of conversation and that's confusing.

This particular issue was a suggestion 6-months ago that was referenced from a request by Marten.

If it's in Koop then we can do all our standard format export, geohash, etc server side in Open data.

**Comment by mhogeweg** [comment]
Server-side processing is ok. Just not the dependency on ArcGIS online. When using portal for ArcGIS there is no reason to not have the utility service also be part of portal for ArcGIS

**Comment by jbartley** [comment]
The GeoRSS reflector service works with online and portal for arcgis.  Portal comes installed with the reflector service.

**Comment by mhogeweg** [comment]
@jbartley we run into situations at customer sites who want to display [GeoRSS in a JS API application](https://developers.arcgis.com/javascript/jssamples/layers_georss.html), but cannot use the online utility service that's in play. Portal for ArcGIS is not necessarily part of the solution, and the RSS feeds are not necessarily public. This is where a possible koop-based solution could help, unless there are ways to configure the JS API with a local utility service outside of the Portal context.

**Comment by mhogeweg** [comment]
small update. I've created a simple JS API layer for GeoRSS that displays (points only currently) items as clusters of points. it is based on a pre-existing cluster layer sample and handles both RSS and ATOM with GeoRSS elements.
